### ISSUE TITLE
How do commonservicelocator and Unity keep in line with the official

### ISSUE THREAD

**Issue by ghd258** [Chavoshi/EnterpriseLibrary.NetCore#9]
https://github.com/Chavoshi/CommonServiceLocator.NetCore
https://github.com/Chavoshi/Unity.NetCore
Is your two address a little duplicated with the official one, the following link address

https://github.com/unitycontainer/unity
https://github.com/unitycontainer/commonservicelocator

If I've already used the official class library of these two addresses, how can I use it with you, or use it at the same time?

**Comment by Chavoshi** [comment]
Hi
The time which I have uploaded these packages there where no official version of Unity and specifically Unity.Interception so I ported the Unity 4.0 libraries all by myself to .Net Core 2.0.
As I checked they still do not ported Unity.Interception (or maybe it's not part of their plan at all) So as I wanted to port the Enterprise Library as well to .Net Standard I needed both packages together.
In the future if they will port Unity.Interception also I will deprecate this packages and use the official ones.
I am going to make use of official Common Service Locator ASAP.
